subsys/bluetooth/services/ras/rrsp/Kconfig.ras_rrsp

```diff
@@ -13,6 +13,40 @@ menuconfig BT_RAS_RRSP
 
 if BT_RAS_RRSP
 
+config BT_RAS_RRSP_AUTO_ALLOC_INSTANCE
+	bool "Automatically allocate RRSP instances to new connections"
+	default y
+
+config BT_RAS_RRSP_MAX_ACTIVE_CONN
+	int "Number of simultaneously supported RRSP instances"
+	default BT_MAX_CONN
+	range 1 BT_MAX_CONN
+	help
+	  The number of simultaneous connections with an instance of RAS RRSP
+
+config BT_RAS_RRSP_RD_BUFFERS_PER_CONN
+	int "Number of ranging data buffers per connection"
+	default 1
+	range 1 10
+	help
+	  The number of ranging procedures that can be stored inside RRSP.
+
+config BT_RAS_RRSP_MAX_SUBEVENTS_PER_PROCEDURE
```

Comment:
Propose replacing BT_RAS_RRSP_MAX_STEPS_PER_PROCEDURE and BT_RAS_RRSP_MAX_STEPS_PER_PROCEDURE with:

- BT_RAS_RRSP_MAX_ANTENNA_PATHS (range 1 4 default 4)
- BT_RAS_RRSP_MODE_3_SUPPORTED (just yes/no, default yes)

then the only thing these configs do is change the value of [BT_RAS_MAX_STEP_DATA_LEN](https://github.com/nrfconnect/sdk-nrf/pull/18750/files#diff-7e73bbe9c26a1eca52dd840500b2ef5659e4965cc1c6a3d405a6779978953db9R57).

We should also make sure there's some kind of warning in the case where the controller generates more data than expected